src/System.Management.Automation/utils/PInvokeDllNames.cs

```diff
@@ -116,7 +116,6 @@ internal static class PinvokeDllNames
         internal const string PeekConsoleInputDllName = "api-ms-win-core-console-l2-1-0.dll";                /*103*/
         internal const string GetNumberOfConsoleInputEventsDllName = "api-ms-win-core-console-l1-1-0.dll";   /*104*/
         internal const string SetConsoleCtrlHandlerDllName = "api-ms-win-core-console-l1-1-0.dll";           /*105*/
-        internal const string SetConsoleCursorPositionDllName = "api-ms-win-core-console-l2-1-0.dll";        /*106*/
```

Comment:
Every line has a number in comment - make sense to update the numbers in the list tail?